app/services/foreman_ansible/playbooks_importer.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module ForemanAnsible
+  # Imports playbooks from smart proxy
+  class PlaybooksImporter
+    include ::ForemanAnsible::ProxyAPI
+    delegate :playbooks, :playbooks_names, :to => :proxy_api
+
+    def initialize(proxy = nil)
+      @ansible_proxy = proxy
+    end
+
+    def import_playbooks(playbooks_names)
+      import_playbooks_content playbooks(playbooks_names)
```

Comment:
This step I'm bit curious about. I guess that we want to phase it into this playbook will be updated, this created, this deleted, but you're already fetching the content of the playbooks for that .
Is it necessary given we operate only on names (and I don't believe we want to do anything else in the future anyway)?

I think it would be better to sort out update/create first with names only and only when we have that sorted (and selected by some input I'd expect to have method that would get names to update, names to create and did what it's told.